scripts/o3de/o3de/export_project.py

```diff
@@ -196,11 +196,13 @@ def get_default_asset_platform():
                                             'darwin':  'mac' }
     return host_platform_to_asset_platform_map.get(platform.system().lower(), "")
 
-def get_platform_installer_folder_name():
+def get_platform_installer_folder_name(selected_platform=None):
+    if not selected_platform:
+        selected_platform = platform.lower()
```

Comment:
Shouldn't this be `selected_platform = CURRENT_PLATFORM` ?
```suggestion
        selected_platform = CURRENT_PLATFORM
```